src/backend/setup.cfg

```diff
@@ -39,11 +39,11 @@ install_requires =
     djangorestframework_simplejwt==4.3.0
     django-safedelete==0.5.1
     # superior versions of django-storages are not compatible with ManifestStaticFilesStorage
-    django-storages==1.7.1
+    django-storages==1.6.3 # pyup: ignore
     dockerflow==2018.4.0
     gunicorn==19.9.0
     # Warning from psycopg2 "The psycopg2 wheel package will be renamed from release 2.8" => psyopg2-binary
-    psycopg2-binary==2.8.2
+    psycopg2-binary==2.7.5 # pyup: ignore
```

Comment:
Add constraints instead of ignoring those dependencies.

Reply:
:ok_hand: